config/initializers/environments.rb

```diff
@@ -1,5 +0,0 @@
-Rails.env.on(:production) do |config|
-  config.log_level = :info
-
-  GA.tracker = 'UA-54767621-1'
-end
```

Comment:
O intuito desses initializers (`config/initializers/environments.rb` e `config/initializers/application.rb`) é não modificar arquivos gerados pelo rails, isso para quando for atualizar a versão do rails não precisa ficar fazendo diff dos arquivos :smile: 
O que acha?